src/__myproject__/_config/config.yaml

```diff
@@ -7,6 +7,11 @@ parameters:
     datalake:
       base_path: '/%kernel.environment%'
 
+    clickhouse:
+      host:
+      port:
+      exclued_columns:
```

Comment:
typo excluded_columns

Reply:
maybe the `excluded_columns` shouldn't be under clickhouse, it should be under something like `export`